### PR TITLE
[Improve] add rust doc

### PR DESCRIPTION
**Pull request by caicancai**
## Related issue number

Closes #xxxx

## Check code requirements

- [ ] tests added / passed (if needed)
- [ ] Ensure all linting tests pass, see [here](https://github.com/alipay/fury/blob/main/CONTRIBUTING.rst) for how to run them

### CODE REVIEW

**Comment by caicancai** [comment]
cc @chaokunyang

**Comment by caicancai** [comment]
@chaokunyang Excuse me, please review my pr if you have time, thank you

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,7 +18,6 @@ mvn -T10 clean test
 ```
 
 ### C++
-Run c++ tests：
```
Comment (chaokunyang) [inline CONTRIBUTING.md:21]: Why remove this?

Comment (chaokunyang) [reply CONTRIBUTING.md:21]: I see. It has been indicated in test section

**Review by chaokunyang — APPROVED** [review]

**Pull request merged**